qa/tasks/lost_unfound.py

```diff
@@ -141,7 +142,13 @@ def task(ctx, config):
                      pg['state']);
             m = manager.list_pg_unfound(pg['pgid'])
             #log.info('%s' % m)
+            out = manager.raw_cluster_cmd('pg', pg['pgid'], 'query')
+            j = json.loads(out)
```

Comment:
https://jenkins.ceph.com/job/ceph-pull-requests/59115/console

`./tasks/lost_unfound.py:146:13: F841 local variable 'j' is assigned to but never used
ERROR: InvocationError for command /home/jenkins-build/build/workspace/ceph-pull-requests/qa/.tox/flake8/bin/flake8 --select=F,E9 --exclude=venv,.tox (exited with code 1)`

Reply:
@neha-ojha I don't see "is assigned to but never used" in the latest run but it still failed.  My fixup! commented out that code.